libtenzir/builtins/formats/zeek_tsv.cpp

```diff
@@ -817,9 +816,8 @@ class zeek_tsv_printer final : public plugin_printer {
       auto array
         = to_record_batch(resolved_slice)->ToStructArray().ValueOrDie();
       auto first = true;
-      auto is_first_schema = static_cast<bool>(*last_schema);
-      auto did_schema_change
-        = *last_schema != input_schema and not is_first_schema;
+      auto is_first_schema = not *last_schema;
+      auto did_schema_change = *last_schema != input_schema;
```

Comment:
```suggestion
      auto schema_changed = *last_schema != input_schema;
```

Why not something simple like that? Even `schema_did_change` would be better. 😉

Reply:
Resolving this without changing anything, as this is stylistic only and not worth checking the branch out locally for again.